plugins/kadalu.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kadalu
+spec:
+  version: "v0.8.7"
+  homepage: https://github.com/kadalu/kadalu
+  shortDescription: "Deploy and manage Kadalu Operator, CSI and Storage pods"
```

Comment:
this is good but it's about 5 chars long than it fits to the "krew search" screen. Maybe delete "Deploy and"

Reply:
Done!